config/sync/core.entity_form_display.node.news.default.yml

```diff
@@ -72,11 +72,12 @@ content:
     settings:
       placeholder_url: ''
       placeholder_title: ''
+      linkit_profile: default
     third_party_settings: {  }
-    type: link_default
+    type: linkit
     region: content
   field_summary:
-    type: text_textarea
```

Comment:
Not sure why field_summary is being changed from text_textarea to string_textarea here - but I think string_textarea is correct